metricflow/test/fixtures/model_yamls/simple_model/metrics.yaml

```diff
@@ -627,7 +627,7 @@ metric:
     - support@transformdata.io
   type: derived
   type_params:
-    expr: (bookings - bookings_2_weeks_ago) / bookings_2_weeks_ago
```

Comment:
Simplified these expressions so we would end up with a more complete set of results